docs/conf.py

```diff
@@ -61,7 +61,8 @@
     'nbsphinx',
     'recommonmark',
     'codediff',
-    'sphinx_markdown_tables'
+    'sphinx_markdown_tables',
```

Comment:
I think this is used to render markdown tables. Maybe try removing it and see if the diffs still render correctly. If so you can remove it.

Reply:
Since we don't know what its being used for maybe lets leave it? Sphinx will not fail if a pluggin is not present.

Reply:
Let's just remove it then, we can always put it back if we find something has broken.